gateway/ln-gateway/src/bin/cln_extension.rs

```diff
@@ -81,13 +81,10 @@ pub struct Htlc {
 
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct Onion {
-    pub payload: String,
-    pub short_channel_id: u64,
+    #[serde(default)]
+    pub short_channel_id: Option<u64>,
```

Comment:
FYI, you'll have to add a custom parsing fn here since we get the scid in `<block>x<tx>x<out_idx>` format and not as a string.

Reply:
ack. pending